tests/population/test_mortality.py

```diff
@@ -33,6 +51,169 @@ def test_mortality_default_lookup_configuration(
     ]
 
     assert set(expected_lookup_table_keys) == set(lookup_tables.keys())
-    assert (lookup_tables["all_cause_mortality_rate"].data["value"] == 0.0).all()
+    assert (lookup_tables["all_cause_mortality_rate"].data["value"] == 0.5).all()
     assert lookup_tables["unmodeled_cause_specific_mortality_rate"].data == 0.0
     assert (lookup_tables["life_expectancy"].data["value"] == 98.0).all()
+
+
+def test_mortality_creates_columns(setup_sim_with_pop_and_mortality):
+    sim, bp, mortality = setup_sim_with_pop_and_mortality
+    pop = sim.get_population()
+    expected_columns_created = set(mortality.columns_created)
+    mortality_created_columns = set(pop.columns).difference(
+        set(bp.columns_created + ["tracked"])
+    )
+    assert expected_columns_created == mortality_created_columns
+
+
+def test_mortality_rate(setup_sim_with_pop_and_mortality):
+    sim, bp, mortality = setup_sim_with_pop_and_mortality
+    sim.step()
+    pop1 = sim.get_population()
+    mortality_rates = mortality.mortality_rate(pop1.index)["other_causes"]
+    # Calculate mortality rate like component to cmpare
+    lookup_tables = mortality.lookup_tables
+    acmr = lookup_tables["all_cause_mortality_rate"](pop1.index)
+    modeled_csmr = mortality.cause_specific_mortality_rate(pop1.index)
+    unmodeled_csmr_raw = lookup_tables["unmodeled_cause_specific_mortality_rate"](pop1.index)
+    unmodeled_csmr = mortality.unmodeled_csmr(pop1.index)
+    expected_mortality_rates = (acmr - modeled_csmr - unmodeled_csmr_raw + unmodeled_csmr) * (
+        sim._clock.step_size.days / 365
+    )
+    # Cannot compare two floats with FuzzyChecker
```

Comment:
Is this something the FuzzyChecker should be able to do?

Reply:
I think if we put in a wrapper that just wraps numpy isclose that would be a good idea but otherwise we would have to develop a whole framework for how to do that.